scripts/kind-with-registry.sh

```diff
@@ -203,34 +203,22 @@ function upload_to_blob() {
 # to be mounted on the kind cluster and hence extra mount flags are required.
 function createKindForAZWI() {
   echo "creating workload-identity-enabled kind configuration"
-  if [ -n "${CONFORMANCE_FLAVOR}" ] && [ -n "${SERVICE_ACCOUNT_SIGNING_PUB}" ] && [ -n "${SERVICE_ACCOUNT_SIGNING_KEY}" ]; then
```

Comment:
> IIRC this condition in the kind script to use a different workload ID setup is a vestige of the pre-community infra and/or pre-workload ID default days and no longer is needed now.

@jackfrancis Does that sound right?

Reply:
That sounds plausible, conformance E2E test runs will tell us!

Reply:
Looks like the tests are all far enough along now that they've all created a cluster successfully, so I think we're in the clear.